### PR TITLE
whitelist more combinations of tags

### LINKED ISSUE #1059: landuse=commercial and amenity is often a valid combination, for example with amenity=boat_storage
https://osmose.openstreetmap.fr/en/map/#item=4030&zoom=17&lat=51.076816&lon=0.384178&level=1&tags=&fixable=
https://www.openstreetmap.org/way/370680096
https://osmose.openstreetmap.fr/en/error/68266df9-dc19-f86f-1938-5912a703fdf3

source | 14449
-- | --
item | 4030
class | 900
lat lon | 51.0768195 0.3841728

### LINKED ISSUE #1060: leisure=playground natural=sand is a valid combination
For better or worse natural=* is used not only for natural features, so complaining about that (as level1 issue) is not useful

See https://www.openstreetmap.org/way/60516744 https://osmose.openstreetmap.fr/en/error/e4e52ae7-c2c1-b3d6-4547-cbe41bdd3bfa https://osmose.openstreetmap.fr/en/map/#&item=4030&zoom=17&lat=35.3038784&lon=-119.0721334&level=1&tags=&fixable= 

source | 38058
-- | --
item | 4030
class | 900
lat lon | 35.3038784 -119.0721334

### LINKED ISSUE #1061: amenity=event_venue leisure=garden is not an invalid combination
https://www.openstreetmap.org/way/606085127 - will be included in soon to be made PR adding exceptions for combinations

At least in Poland there are many gardens that are used as event venues - especially for weddings

### LINKED ISSUE #1062: more valid combinations
https://www.openstreetmap.org/way/129534572 amenity=monastery landuse=religious is a fine combination

https://www.openstreetmap.org/way/102243623 amenity=fuel landuse=retail

https://www.openstreetmap.org/node/6050451085 Conflict between tags: amenity, leisure - amenity=gambling leisure=adult_gaming_centre

### PR DESCRIPTION
**Pull request by matkoniecz**
fixes #1062 fixes #1061 fixes #1060 fixes #1059

### CODE REVIEW

**Comment by matkoniecz** [comment]
warning: I have not tested this code

**Comment by matkoniecz** [comment]
```
FAILED plugins/TagFix_Brand.py::Test::test_CA - KeyError: 'name'

FAILED plugins/TagFix_Brand.py::Test::test_FR - KeyError: 'name'
```
Failure seems unrelated.

File: plugins/TagRemove_Incompatibles.py [hunk 1]
```diff
@@ -63,12 +66,15 @@ def init(self, logger):
...
                 ['water', 'amenity', 'fountain'], # ?
+                ['sand', 'leisure', 'playground'],
             ],
             'amenity': [
                 ['stables', 'leisure', 'horse_riding'],
                 ['drinking_water', 'natural', 'spring'],
                 ['drinking_water', 'man_made', 'water_tap'],
                 ['shelter', 'highway', 'bus_stop'],
+                ['event_venue', 'leisure', 'garden'],
+                ['gambling', 'leisure', 'adult_gaming_centre'],
```
Comment (frodrigo) [inline plugins/TagRemove_Incompatibles.py:77]:
```suggestion
```
```suggestion
                ['gambling', 'leisure', 'adult_gaming_centre'],
```

**Comment by frodrigo** [comment]
Thank you

**Comment by matkoniecz** [comment]
And thanks for a quick review and merging most of them!

What was wrong with `['gambling', 'leisure', 'adult_gaming_centre'],` part? It also seems like a clear false positive.

(I ask as I more likely to submit more and I would prefer to not send ones that will be rejected anyway)

**Comment by frodrigo** [comment]
You are right. This last one is OK too.
Please make an other PR with it.

**Pull request merged**